### PR TITLE
Completed javascript-methods Exercise - Bingle Padilla

### PR DESCRIPTION
**Pull request by codewithbingle**
javascript-methods

### CODE REVIEW

File: javascript-methods/main.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+};
+
+var css = {
+  title: ' CSS Secrets',
+  author: 'Lea Verou'
+};
+
+library.push(js);
+library.unshift(css);
+library.splice(2, 1);
```
Comment (Zhiwutian) [inline javascript-methods/main.js:47]: this should be 1 and 1, and two books should remain.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
tiny update needed!

File: javascript-methods/main.js (hunk 1 above)
Comment (codewithbingle) [reply javascript-methods/main.js:47]: Ok got it.

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Comment by codewithbingle** [comment]
javascript-methods-update2

**Pull request merged**